cmd/auth/main.go

```diff
@@ -43,9 +47,62 @@ func main() {
 
 	setupTasks(p.TaskExecutor, p)
 
-	server := httputil.NewServer(p.LoggerFactory, &http.Server{
-		Addr:    serverCfg.ListenAddr,
+	publicServer := &http.Server{
+		Addr:    serverCfg.PublicListenAddr,
 		Handler: setupRoutes(p, configSource),
-	})
-	server.ListenAndServe("starting auth gear")
+	}
+
+	internalServer := &http.Server{
```

Comment:
I think some may want to run two separate deployment for public & internal server, so may be add a flag to control which server(s) is started?

Reply:
Maybe it should be a command argument? Delay it until we do https://github.com/authgear/authgear-server/issues/11 ?